README.md

```diff
@@ -52,6 +53,9 @@ application:
         res.redirect('/');
       });
 
+**Optional Parameters:**
+- **passReqToCallback** (default false) - directs passport to send the request object to the verfication callback
+
 ## Examples

```

Comment:
Could you remove the examples and test sections here?

Reply:
done